nixos/release.nix

```diff
@@ -104,6 +104,22 @@ in rec {
   initialRamdisk = buildFromConfig ({ pkgs, ... }: { }) (config: config.system.build.initialRamdisk);
 
 
+  netboot = let build = (import lib/eval-config.nix {
+      system = "x86_64-linux";
+      modules = [
+        ./modules/installer/netboot/netboot-minimal.nix
+        versionModule
+      ];
+    }).config.system.build;
+  in
+    pkgs.symlinkJoin "netboot" [
+      build.netbootRamdisk
+      build.kernel
+      build.netbootKernelAppend
+      build.netbootTestScript
```

Comment:
Not sure if `netbootTestScript` should be removed. With the boot test it isn't as useful, but it might still be practical for some.

I do think `netbootKernelAppend` can be removed, as there is no use for that anymore that I can think of.